app/jobs/seed_users_data_job.rb

```diff
@@ -27,6 +27,9 @@ class SeedUsersDataJob
 
   SYNC_PAYLOAD_SIZE = 20
 
+  SEED_DATA_AGE = 24.months
```

Comment:
Non-blocking: I think these constants are ok, but the intention of the traits is to really have whatever curve you want to build for the various traits, so pulling them out to a couple of numbers isn't really that valuable, even if they are mostly the same for now.